docs/webadministration.rst

```diff
@@ -0,0 +1,436 @@
+For more information see the section 'Admin account - automatic creation' in :ref:`the configuration reference <admin_account>`.
+
+The following sections are only accessible for global administrators:
+
+* send broadcast messages to all users (Menu item Broadcasts)
+
+* configure global administration users (Menu item Administrators)
+
+* configure relayed domains (Menu item Relayed domains)
+
+* access Rspamd webui (Menu item Antispam)
+
+* Configure all email domains served by Mailu (Menu item Mail domains)
+
+
+.. _webadministration_settings:
+
+Settings
+--------
+After logging in the web administration interface, the settings page is loaded. 
+On the settings page the settings of the currently logged in user can be changed.
+Changes are saved and effective immediately after clicking the Save Settings button at the bottom of the page.
+
+
+Display name
+````````````
+
+On the settings page the displayed name can be changed of the logged in user. 
+This display name is only used within the web administration interface.
+
+
+Antispam
+````````
+
+Under the section `Antispam` the spam filter can be enabled or disabled for the logged in user. By default the spam filter is enabled. 
+
+When the spam filter is disabled, all received email messages will go to the inbox folder of the logged in user.
+The exception to this rule, are email messages with an extremely high spam score. These email messages are always rejected by Rspamd.
+
+When the spam filter is enabled, received email messages will be moved to the logged in user's inbox folder or junk folder depending on the user defined spam filter tolerance.
+
+The user defined spam filter tolerance determines when an email is classified as ham (moved to the inbox folder) or spam (moved to the junk folder). 
+The default value is 80%. The lower the spam filter tolerance, the more false positives (ham classified as spam). The higher the spam filter tolerance, the more false negatives (spam classified as ham). 
+For more information see the :ref:`antispam documentation <antispam_howto>`.
+
+Auto-forward
+`````````````
+Under the section `Auto-forward`, the automatic forwarding of received email messages can be enabled. When enabled, all received email messages are forwarded to the specified email address. 
+
+The option "Keep a copy of the emails" can be ticked, to keep a copy of the received email message in the inbox folder.
+
+In the destination textbox, the email addresses can be entered for automatic forwarding. When entering multiple email addresses the comma (',') must be used as delimiter.
+
+
+Update password
+---------------
+
+On the `update password` page, the password of the logged in user can be changed. Changes are effective immediately. 
+
+
+.. _webadministration_auto-reply:
+
+Auto-reply
+----------
+
+On the `auto-reply` page, automatic replies can be configured. This is also known as out of office replies.
```

Comment:
```suggestion
On the `auto-reply` page, automatic replies can be configured. This is also known as out of office (ooo) or out of facility (oof) replies.
```

Especially the latter is often used in Microsoft contexts, so I think it makes sense to add them in case someone searches for them with ctrl+f